dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry1.1.ee-8.0.feature

```diff
@@ -0,0 +1,21 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpTelemetry1.1.ee-8.0
```

Comment:
Instead of ee-8.0, there should be a:
mp-2.0 with mpConfig-1.3 and mpRestClient-1.1
mp-2.2 with mpConfig-1.3 and mpRestClient-1.2
mp-3.0 with mpConfig-1.3 and mpRestClient-1.3
mp-3.3 with mpConfig-1.4 and mpRestClient-1.4
mp-4.0 with mpConfig-2.0 and mpRestClient-2.0

The mp-4.0 private feature will have mpCompatible-4.0 and the others will have mpCompatible-0.0

I would recommend putting all of the non mpConfig, mpRestClient and mpCompatible feature and bundle dependencies into its own private feature that the mp-x.y ones above would depend on.  You could call that private feature ee-8.0 if you wanted to since all of the ee-8.0 dependencies will be in that feature.

Reply:
Scope has been reduced so that we're only targeting one version of MicroProfile for each of EE 7, 8 and 9, so the current feature layout should be fine.